packages/style-guide/src/context.js

```diff
@@ -1,4 +1,4 @@
 import { useContext } from 'react'
 import { ThemeContext } from '@emotion/core'
 
-export const useTheme = () => useContext(ThemeContext)
+export const useThemeUI = () => useContext(ThemeContext)
```

Comment:
Why rename this? This isn't the same thing as `useThemeUI`

Reply:
Yes sorry about that. I've removed those changes